_drafts/2017-05-16-lingtypology.Rmd

```diff
@@ -0,0 +1,199 @@
+
+## 1. Installation
+
+The package is available via [CRAN](https://CRAN.R-project.org/package=lingtypology):
+```{r, eval = FALSE}
+install.packages("lingtypology", dependencies = TRUE)
+```
+
+However, as the package is still under construction, I would suggest users to download the development version instead, which is available at Github and can be accessed using [devtools](https://CRAN.R-project.org/package=devtools):
+
+```{r, eval=FALSE}
+devtools::install_github("ropensci/lingtypology")
+```
+
+Since `lingtypology` depends on the [`leaflet` package](https://rstudio.github.io/leaflet/), it will work only with R version greater than or equal 3.1.0.
+
+Load the package:
+```{r}
+library("lingtypology")
+```
+
+## 2. Simple maps
+The main function for creating maps is `map.feature()`. Let's create a map with some languages:
+
+```{r}
+map.feature(languages = c("Irish", "Danish", "Rusyn", "Bebe"))
+```
+By using just one line we have created an interactive leaflet map. Each point on the map has a pop-up box that appears when the marker is clicked. The pop-up box by default contains the name of the language with a link to its page on Glottolog.  
+There are several functions in `lingtypology` for finding a language by affiliation, ISO-code or Glottolog code. For example, if you are interested in Lezgic languages and don't want to list all of them separately, you can access them through the database with the following function:
+```{r}
+lang.aff("Lezgic")
+```
+Most of the functions in `lingtypology` have the same syntax: **what you need.what you have**. So you can search for additional information such as area, country name, ISO code or glottolog code using the same syntax. See the [package manual](https://ropensci.github.io/lingtypology/#2_glottolog_functions) for a full list of functions.
+```{r}
+area.lang(c("Bebe", "Irish")) #area
+iso.lang(c("Adyghe", "Koasati")) # ISO code
+gltc.lang(c("Nihali", "Malali")) # Glottolog code
+```
+
+Using these functions you can do incredible things and easily create maps that haven't been created before. Here is my favourite map with showing all Sign languages from the database.
+
+```{r}
+map.feature(lang.aff("Sign"))
+```
+
+If you have a map with a lot of points it can be tiresome to have to click on each point in order to know which language it respresents. For this purpose, the 'label' argument can be used: it labels each point with a certain parameter, for example the name of the languages. 
+
+```{r}
+map.feature(lang.aff("Sign"), label = lang.aff("Sign"))
+```
+
+On this map, the labels are displayed on mouse over, but it is also possible to make the labels visible at all times. In this case I used %the same vector as a language set, but it is possible to use any other vector. %вот это я не поняла?
```

Comment:
I'm not sure what this means: "%the same vector". If it's not an error, perhaps clarify for a more novice reader like me.